templates/data/mysql/support.html

```diff
@@ -0,0 +1,632 @@
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/7d0d3928-opensearch.png",
+                            alt="",
+                            width="568",
+                            height="320",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-equal-height-row__item">
+            <p>Learn more&nbsp;&rsaquo;</p>
+          </div>
+        </a>
+
+        <a class="p-equal-height-row__col"
+           href="/data/kafka"
+           aria-label="Learn more about Kafka">
+          <div class="p-equal-height-row__item">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/862a183d-kafka.png",
+                            alt="",
+                            width="568",
+                            height="320",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-equal-height-row__item">
+            <p>Learn more&nbsp;&rsaquo;</p>
+          </div>
+        </a>
+
+        <a class="p-equal-height-row__col"
+           href="/data/postgresql"
+           aria-label="Learn more about PostgreSQL">
+          <div class="p-equal-height-row__item">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/1c1126a6-postgres.png",
+                            alt="",
+                            width="568",
+                            height="320",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-equal-height-row__item">
+            <p>Learn more&nbsp;&rsaquo;</p>
+          </div>
+        </a>
+
+        <a class="p-equal-height-row__col"
+           href="/data/mongodb"
+           aria-label="Learn more about MongoDB">
+          <div class="p-equal-height-row__item">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/35fd8102-mongodb.png",
+                            alt="",
+                            width="568",
+                            height="320",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-equal-height-row__item">
+            <p>Learn more&nbsp;&rsaquo;</p>
```

Comment:
Since there is no text here we can add `aria` labels for more accessibility.

Reply:
I believe that won't be necessary since the parent element already has an aria label:
```html
        <a class="p-equal-height-row__col"
           href="/data/mongodb"
           aria-label="Learn more about MongoDB">
```

Let me know if you still think whether it should be added.

Reply:
You are right, I missed that.
It is not needed then